src/ServiceManagement/StorSimple/Commands.StorSimple/ServiceClients/PSStorSimpleDevicesClient.cs

```diff
@@ -0,0 +1,47 @@
+﻿
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Azure;
+using Microsoft.WindowsAzure.Commands.StorSimple;
+using Microsoft.WindowsAzure.Management.StorSimple;
+using Microsoft.WindowsAzure.Management.StorSimple.Models;
+
+namespace Microsoft.WindowsAzure.Commands.StorSimple
```

Comment:
We were suggested to use WindowsAzure instead of Azure in all namespaces and dll names, has there been any recent change in policy?